news/2019-01-30-new-changes-to-star-rating-performance-points.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "New Changes to Star Rating and Performance Points"
+date: 2019-01-30 01:00:00 +0000
+---
+
+After many years of stagnancy, some changes are finally coming to the way Star Rating and Performance Points are calculated! Read on to find out more.
+
+![](temporary.banner)
+
+Hello!
+
+It is with great anticipation that I, Xexxar, am here to announce changes to both Performance Points (aka: pp) and the star rating system. 
+
+I have been closely involved with various other members of the community over the past few months in proposing amendments to the Performance Points system with the end goal of updating a core part of osu!'s competitive progression that hasn't seen any significant changes in many, many years. The game has evolved considerably since the last update to these key systems, with the introduction of many new and different styles of mapping and gameplay.
+
+While the updates we've just made are not designed to fix *every* issue with the systems as they currently stand, we hope that the changes will allow players to better assess skills that were formerly not factored in properly in previous versions of the algorithm.
+
+Without further delay, let's get into the specific changes:
+
+## Wide Angle Jumps
+
+As many players are well aware, the angle of a jump impacts how difficulty a pattern is to combo. Evidence of this can be found all throughout modern mapping. Almost all maps that are popular for their PP values contain jumps that are typically non-linear. 
+
+As such, through the usage of a new angle calculation, a new scaling reward for jumps with angles exceeding 60 degrees has been added. 
+
+Decided not to farm a map because it had square jumps? Found yourself baffled by the difficulty of an older 2013 map with linear flow? These previously underweighted maps now have a more accurate star rating associated with them (this includes alt maps). And yes, they will be worth more PP as a result.
```

Comment:
```suggestion
Decided not to farm a map because it had square jumps? Found yourself baffled by the difficulty of an older 2013 map with linear flow? These previously underweighted maps now have a more accurate star rating associated with them (this includes alt maps). Also, yes, they will be worth more PP as a result.
```

Reply:
I think this is fine the way it is.